core/src/consensus/stake/mod.rs

```diff
@@ -168,6 +204,39 @@ pub fn get_stakes(state: &TopLevelState) -> StateResult<HashMap<Address, u64>> {
     Ok(result)
 }
 
+fn change_params(
+    state: &mut TopLevelState,
+    metadata_seq: usize,
+    params: CommonParams,
+    signatures: &[Signature],
+) -> StateResult<()> {
+    state.update_params(metadata_seq, params.clone())?;
+
+    let action = Action::ChangeParams {
+        metadata_seq,
+        params,
+        signatures: vec![],
+    };
+    let encoded_action = H256::blake(rlp::encode(&action));
+    let stakes = get_stakes(state)?;
+    let signed_stakes = signatures.iter().try_fold(0, |sum, signature| {
+        let public = recover(signature, &encoded_action).unwrap_or_else(|err| {
+            unreachable!("The transaction with an invalid signature cannot pass the verification: {}", err);
+        });
+        let address = public_to_address(&public);
+        stakes.get(&address).map(|stake| sum + stake).ok_or_else(|| RuntimeError::SignatureOfInvalidAccount(address))
+    })?;
+    let total_stakes: u64 = stakes.values().sum();
+    if total_stakes / 2 > signed_stakes {
```

Comment:
https://github.com/CodeChain-io/codechain/pull/1530/files#diff-ec496aca1d063d60bc2a7e03e1bcea3dR385
Half of the stakeholders? Half of the stakes?

Reply:
They mean the same thing.
Mostly, if there are no mentions about it, one stake means one vote.
https://en.wikipedia.org/wiki/One_share,_one_vote